### DISCUSSION TITLE
Docs been strange when I try to click the APIs

### DISCUSSION THREAD

**Discussion by raymond-liao** [fastapi/fastapi#7374] [Questions]
# ENV:
Python: 3.7.5
fastapi: 0.55.1
Google Chrome: Version 84.0.4147.89 (Official Build) (64-bit)

# The video record:
![record](https://user-images.githubusercontent.com/10546774/88247884-f3e00f00-ccd1-11ea-888e-432551d3a8bc.gif)

Please help.

**Comment by phy25** [comment]
Duplicate of #1770.

**Comment by raymond-liao** [comment]
@phy25 thanks, I'll make a try.

**Comment by raymond-liao** [comment]
@phy25 it's working after the FastAPI is updated to 0.60.1, thanks.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :clap: :bow: 

Thanks for reporting back and closing the issue @gaoshanyu  :+1:
